modules/lazyload/module.php

```diff
@@ -42,34 +42,62 @@ private function update_element_attributes( Element_Base $element ) {
 		$settings = $element->get_settings_for_display();
 		$controls = $element->get_controls();
 		$lazyload_attribute_name = 'data-e-bg-lazyload';
-
+		$attributes = [];
 		$controls_with_background_image = array_filter( $controls, function( $control ) {
 			return Utils::get_array_value_by_keys( $control, [ 'background_lazyload', 'active' ] );
 		} );
 
+		$repeater = array_filter( $controls, function( $control ) {
```

Comment:
Extract this code into functions, what is the input and the output of the function?

Reply:
controls -> repeater

Done